test_utils/filesystem/symlink.py

```diff
@@ -21,3 +23,11 @@ def md5sum(self, binary=True):
 
     def get_target(self):
         return readlink(self.full_path)
+
+    @staticmethod
+    def new_symlink(link_name, path, target, force: bool = False):
+        if not check_if_directory_exists(path):
+            create_directory(path, True)
+        full_path = os.path.join(path, link_name)
+        TestRun.executor.run(f"ln -s {target} {full_path} {'-f' if force else ''}")
```

Comment:
We need a successful execution to return Symlink, so:
```suggestion
        TestRun.executor.run_expect_success(f"ln -s {target} {full_path} {'-f' if force else ''}")
```